src/closure.ts

```diff
@@ -88,7 +88,10 @@ export default class Closure extends Callable {
     if (this.node.type === 'FunctionDeclaration' && this.node.id !== null) {
       this.functionName = this.node.id.name
     } else {
-      this.functionName = `Anonymous${++Closure.lambdaCtr}`
+      this.functionName =
+        `Anonymous${++Closure.lambdaCtr} (` +
+        this.node.params.map(o => Object(o).name).join(', ') +
```

Comment:
```suggestion
        this.node.params.map((o: es.Identifier) => o.name).join(', ') +
```

Let's try to use types as much as possible